spec/index.html

```diff
@@ -1498,15 +1496,15 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
 
       <dt id="JSON-mapping">The <a>lexical-to-value mapping</a></dt>
       <dd>
-        maps every element of the lexical space to the result of parsing it into an
-        <a data-cite="JSON-LD11#dfn-internal-representation">internal representation</a>
+        maps every element of the lexical space to the result of parsing it into a
+        <a data-cite="RFC4627#section-3">JSON value</a>
```

Comment:
Wrong reference? Section 3 is "Encoding"

Reply:
```suggestion
        <a data-cite="RFC4627#section-2.1">JSON value</a>
```